docs/development/getting-started.rst

```diff
@@ -0,0 +1,96 @@
+===============
+Getting Started
+===============
+
+We’re pleased that you are interested in working on pip.
+
+This document is meant to get you setup to work on pip and to act as a guide and
+reference to the the development setup. If you face any issues during this
+process, please `open an issue`_ about it on the issue tracker.
+
+Development tools
+=================
+
+pip uses :pypi:`tox` for testing against multiple different Python environments
+and ensuring reproducible environments for linting and building documentation.
+
+For developing pip, you need to install ``tox`` on your system. Often, you can
+just do ``python -m pip install tox`` to install and use it.
+
+Running Tests
+-------------
+
+pip uses the :pypi:`pytest` test framework, :pypi:`mock` and :pypi:`pretend`
+for testing. These are automatically installed by tox for running the tests.
```

Comment:
Should we also say "and various pytest plugins" here?

Reply:
IMO no, we don't need to complicate things by adding that much detail.